cdk/lib/cdk-stack.ts

```diff
@@ -52,16 +52,12 @@ export class CdkStack extends GuStack {
       description: "Lambda to notify about instances with missing tags",
       timeout: Duration.seconds(30),
       memorySize: 512,
-      vpc: GuVpc.fromId(this, "vpc", parameters.vpc.valueAsString),
+      vpc: GuVpc.fromId(this, "vpc", { vpcId: parameters.vpc.valueAsString }),
       vpcSubnets: {
         subnets: GuVpc.subnets(this, parameters.subnets.valueAsList),
       },
-      rules: [
-        {
-          schedule: Schedule.rate(lambdaFrequency),
-          description: `Run tag-janitor every ${lambdaFrequency.toHumanString()}`,
-        },
-      ],
+      schedule: Schedule.rate(lambdaFrequency),
+      monitoringConfiguration: { noMonitoring: true },
```

Comment:
This PR is a simple upgrade, so I won't add new resources here; I will add an alert via monitoringConfiguration in a subsequent PR.